code/game/objects/items/tanks/watertank.dm

```diff
@@ -169,7 +169,38 @@
 	return new /obj/item/reagent_containers/spray/mister/janitor(src)
 
 /obj/item/reagent_containers/spray/mister/janitor/mode_change_message(mob/user)
-	to_chat(user, span_notice("You [amount_per_transfer_from_this == 10 ? "remove" : "fix"] the nozzle. You'll now use [amount_per_transfer_from_this] units per spray."))
+	to_chat(user, span_notice("You [amount_per_transfer_from_this == 10 ? "remove" : "affix"] the nozzle. You'll now use [amount_per_transfer_from_this] units per spray."))
+
+//Security tank
+/obj/item/watertank/pepperspray
+	name = "ANTI-TIDER-2500"
```

Comment:
Much as the name is wonderful, it doesn't actually tell you what it is. Could you spare a `ANTI-TIDER-2500 supression backpack`?

Not blocking though, if you disagree.

Reply:
You're absolutely correct. Updated